src/slackapp/event-listeners/reimbursementReply.js

```diff
@@ -18,17 +19,31 @@ const filterAndReply = async (event) => {
   if (event.bot_id) {
     return;
   }
-  const reimbursementChannel = await findChannelByName(REIMBURSEMENT_CHANNEL);
-  if (!isReimbursementReply(event, reimbursementChannel.id)) {
+
+  let reimbursementChannel;
+
+  try {
+    reimbursementChannel = await findChannelByName(REIMBURSEMENT_CHANNEL);
+    if (!isReimbursementReply(event, reimbursementChannel.id)) {
+      return;
+    }
+  } catch (e) {
+    console.error("Error finding reimbursement channel", e);
```

Comment:
quick question, non blocking for PR: do console.error logs show up on our application runtime logs on Heroku?

Reply:
I'm actually not sure... I used console.error here because I saw it elsewhere in the codebase, didn't think about it much beyond that.

Reply:
they do! and they stand out a bit more